aspnet/migration/rc1-to-rc2.rst

```diff
@@ -1,22 +1,22 @@
-Migrating from ASP.NET 5 RC1 to ASP.NET Core
-============================================
+Migrating from ASP.NET 5 RC1 to ASP.NET Core 1.0
```

Comment:
That's a bit confusing. Migrating from RC2 involves almost no changes. Maybe I should write a topic on migrating from RC2?